manual/Makefile

```diff
@@ -44,8 +44,10 @@ endif
 qclean:
 	$(LATEXMK) -C $(NAME).tex
 
-install: all
-	@echo "$(NAME).pdf is ready to use, no need to install it."
+install: xtp_manual.pdf
+	mkdir -p $(XTP_MANUAL)
+	cp $< $(XTP_MANUAL)/xtp_manual.pdf
+	echo "Installing to ${XTP_MANUAL}"
```

Comment:
That will break packaging, which uses the `DESTDIR` variable. Use
```
	mkdir -p $(DESTDIR)$(XTP_MANUAL)
	cp $< $(DESTDIR)$(XTP_MANUAL)/xtp_manual.pdf
```
and test with:
```
$ make manual
$ make installmanual DESTDIR=/tmp
```